### ISSUE TITLE
Reduce A-10 starting fuel from 100% to 60%

### ISSUE THREAD

**Issue by dharmab** [Kilcekru/dcc#118]
100% fuel in the A-10 is 3 hours of flight time, which is not needed in campaigns such as Red Bullet where flights are more likely to last 45-90 minutes. The A-10 is much more maneuverable when the fuel is loaded to 60% instead. Indeed, many multiplayer servers start A-10s at 50% or 60% fuel for this reason.

**Comment by Dragonius** [comment]
"Indeed, many multiplayer servers start A-10s at 50% or 60% fuel for this reason." which ones ?

**Comment by dharmab** [comment]
Heatblur Cold War uses 50%, for example

**Comment by Dragonius** [comment]
You’re right—on smaller maps, it would make sense to have slightly less fuel. However, for larger maps like Syria, Afghanistan, or Iran, where we can actually fly for a full three hours, a full tank is ideal.

Sometimes, reaching a CAS or strike mission can take time, especially if there are friendlies and enemies already in the area, and the airspace needs to be cleared first.

I’m not sure what direction the developers of this DCC are taking, but I’d really appreciate an option to adjust the fuel load before the mission is generated. That way, players could customize it based on how long they’d like to fly.

I didn’t mean to come across as rude—it’s just that I don’t play on many servers, and it can be tricky to check the loadout of every plane on every server. Thanks for sharing this info about the “Heatblur Cold War” server—it’s much appreciated!
